src/cascadia/CascadiaPackage/Package-Dev.appxmanifest

```diff
@@ -30,7 +30,20 @@
   </Dependencies>
 
   <Resources>
-    <Resource Language="x-generate"/>
+    <Resource Language="EN-US" />

```

Comment:
does dev still work even if we don't have resources for these languages?

Reply:
woof, didn't test that. I will test the scenarios after lunch.